include/bpf_helper_defs.h

```diff
@@ -339,3 +339,78 @@ EBPF_HELPER(int32_t, bpf_is_current_admin, (const void* ctx));
 #ifndef __doxygen
 #define bpf_is_current_admin ((bpf_is_current_admin_t)BPF_FUNC_is_current_admin)
 #endif
+
+/**
+ * @brief Copy memory from one location to another.
+ *
+ * @param[in] destination Destination buffer.
+ * @param[in] destination_size Size of the destination buffer.
+ * @param[in] source Source buffer.
+ * @param[in] source_size Size of the source buffer.
+ *
+ * @retval 0 The operation was successful.
+ * @retval -EINVAL One or more parameters are invalid.
+ */
+EBPF_HELPER(long, bpf_memcpy, (void* destination, uint32_t destination_size, const void* source, uint32_t source_size));
+#ifndef __doxygen
+#define bpf_memcpy ((bpf_memcpy_t)BPF_FUNC_memcpy)
+#endif
+
+/**
+ * @brief Compare two memory regions.
+ *
+ * @param[in] memory1 First memory region.
+ * @param[in] memory1_size Size of the first memory region.
+ * @param[in] memory2 Second memory region.
+ * @param[in] memory2_size Size of the second memory region.
+ *
+ * @returns 0 if the memory regions are equal, a negative value if memory1 is less than memory2, or a positive value if
```

Comment:
I think you mean if the _contents_ of memory1 is less than the _contents_ of memory2, etc. (not the pointer value itself)

Reply:
Thanks! These are actually part of #3324 but those changes make process monitoring programs easier to write. I will address the comments in the other PR and merge into this one.